processccd.rst

```diff
@@ -69,32 +72,36 @@ Entrypoint
 Butler Inputs
 =============
 
-The main method, ``run``, takes a single butler data reference for the raw input data.
+The main method, ``run``, takes a single butler data reference for the ``raw`` input data.
+
+Butler Outputs
+==============
 
 Examples
 ========
 
+The ``obs_test`` package  models a simple camera with one CCD and includes a data repository containing a few raw images (simulating three visits, two with with the `g`-band filter, and one with the `r`-band one), and some associated calibration data. Its camera consists of a single CCD whose geometry matches a subregion of a single LSST CCD.
```

Comment:
Have added a bit of info about what's actually in obs_test, though we link to it and they can go there for details as needed.